pages/common/nload.md

```diff
@@ -0,0 +1,12 @@
+# nload
+
+> A tool for visualizing network usage in the terminal.
+> More information: <https://github.com/rolandriegel/nload>.
+
+- View all network traffic (use the arrow keys to switch interfaces):
+
+`nload`
+
+- View network traffic on a particular interfaces (use the arrow keys to switch interfaces):
```

Comment:
```suggestion
- View the network traffic on specific interfaces (use the arrow keys to switch interfaces):
```

Reply:
Sure. Is this a general rule or just in this example?

Reply:
Just in this example 👍🏻

Reply:
If you feel strongly otherwise, we don't need at all.